src/main/java/com/example/gildedrose/item/Updater.java

```diff
@@ -1,7 +1,9 @@
-public class GildedRose {
+package com.example.gildedrose.item;
+
+public class Updater {
     public Itemable[] items;
 
-    public GildedRose(Itemable[] items) {
+    public Updater(Itemable[] items) {
```

Comment:
**suggestion (non-blocking)** Personally, I tend to avoid array literals in Java. I prefer to use [collections](https://docs.oracle.com/javase/tutorial/collections/interfaces/collection.html) because the API is richer and at a higher abstraction level.

This would enable you to some thing like the following

```java
class Updater {
    Updater(List<Itemable> items) {
        // ...
    }

    public void updateQuality() {
        items.stream().map(Itemable::update) // 🙀 
    }
}
```

Again, this could be something to consider in a later change.

Reply:
Makes sense.  Will incorporate.